common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -662,6 +688,32 @@ def get_problem_html(self, encapsulate=True):
 
         return html
 
+    def remove_extra_tags(self, html):
+        """
+        The capa xml includes many tags such as <additional_answer> or <demandhint> which are not
+        meant to be part of the client html. We strip them all and return the resulting html.
+        """
+        tags = ['demandhint', 'choicehint', 'optionhint', 'stringhint', 'numerichint', 'optionhint',
```

Comment:
In the old days, I would make it a constant. But with time I've moved my vote that if something is just used in one place, just define it right there. We can make it a constant if you prefer, but my vote would be to just leave it here.